public/style.css

```diff
@@ -37,3 +38,64 @@ h2 {
   margin: 5vh 15vw;
   text-align: center;
 }
+
+/* flex stuff   */
+
+figure img{
+  width: 200px;
+}
+.header{
+  border:1px solid black;
+  margin-top: 5px;
+  margin-bottom: 5px;
```

Comment:
@drainpip true, I should have specified that not all uses of shorthand are beneficial. From my experience, margin padding and border have been easier to read in shorthand. Could also just be due to my own experience as well. Background is definitely not fun to read when shorthand though.